notebook/modelling/experiment_design.ipynb

```diff
@@ -177,13 +177,18 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
+<<<<<<< HEAD
```

Comment:
Looks like you have some merge conflicts here. Delete the notebooks and rebuild.